handlers/deployment.rb

```diff
@@ -32,7 +32,7 @@ class Deployment < Handler
       route(/^(status|build|migrate|deploy|lock|unlock)/, :reversed, command: true)
 
       route(/^panoptes (status|version)/, :status, command: true, help: {"panoptes status" => "Returns the number of commits not deployed to production."})
-      route(/^(panoptes) update tag/, :update_tag, command: true, help: {"panoptes update tag" => "Triggers a GitHub production tag update via Jenkins."})
+      route(/^(panoptes) update tag(\sand build)?$/, :update_tag, command: true, help: {"panoptes update tag" => "Triggers a GitHub production tag update via Jenkins, in turn dockerhub & Jenkins will build a new production AMI."})
       route(/^(panoptes) build/, :build, command: true, help: {"panoptes build" => "Triggers a build of a new AMI of *PRODUCTION* in Jenkins."})
```

Comment:
Should we remove the `build` command?

Reply:
I thought we may want to manually rebuild an image... maybe if the webhook between dockerhub -> jenkins fails?? better to have this and use it on occasion than remove it i thought.